Balode_Inga_dz_1

inga-balode opened the pull request.

inga-balode committed:
Домашнее задание по первому уроку

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,72 @@
+sum = 0
+
+for a in cubed_lst:
+    a = int(a)
```
операция int() лишняя, в списке cubed_lst хранятся как раз целые числа.

inga-balode replied:
Да, верно, изначально создала список переобразовав числа в строчные значения. Потом поправила, а операцию int() в дальнейшем коде убрать забыла. Спасибо!

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,72 @@
+
+for a in cubed_lst:
+    a = int(a)
+    if a % 7 == 0:
```
по условию здесь не само число должно делиться на 7, а сумма его цифр

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,72 @@
+    if a % 7 == 0:
+        while a > 0:
+            digit = a % 10
+            sum = sum + digit
```
нужно вычислять сумму чисел, а не цифр всех этих чисел

inga-balode replied:
Спасибо! Невнимательно прочитала задание.

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,72 @@
+new_cubed_list = []
+
+for x in cubed_lst:
+    x = int(x) + 17
```
операция int() лишняя, в списке cubed_lst хранятся как раз целые числа.

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,72 @@
+
+new_summ = 0
+for x in new_cubed_list:
+    x = int(x)
```
операция int() лишняя, в списке new_cubed_list хранятся как раз целые числа.

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,72 @@
+#     cubed_lst[id] += 17
+
+for i in cubed_lst:
+    i = int(i)
```
операция int() лишняя, в списке cubed_lst хранятся как раз целые числа.

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,72 @@
+n = 0
+summ = 0
+while n < len(cubed_lst):
+    cubed_lst[n] = int(cubed_lst[n]) + 17
```
операция int() лишняя, в списке cubed_lst хранятся как раз целые числа.

inga-balode committed:
Update DZ_1._2.py

Были вопросы к формулировке задания, поэтому сделала 2 варианта (сумма чисел с учетом прибавления 17 и без)

damik21 commented on Balode_Inga_dz_1/DZ_1._2.py:
```diff
@@ -0,0 +1,104 @@
+print(cubed_lst)
+
+sum = 0
+result = 0
```
Здесь как и в остальных местах переменную `result` вне цикла можно не определять
